forge-gui/res/cardsfolder/c/captured_by_the_consulate.txt

```diff
@@ -4,6 +4,6 @@ Types:Enchantment Aura
 K:Enchant creature you don't control
 A:SP$ Attach | Cost$ 3 W | ValidTgts$ Creature.YouDontCtrl | AILogic$ Curse
 S:Mode$ Continuous | Affected$ Creature.EnchantedBy | AddHiddenKeyword$ CARDNAME can't attack. | Description$ Enchanted creature can't attack.
-T:Mode$ SpellCast | ValidCard$ Card | ValidActivatingPlayer$ Opponent | IsSingleTarget$ True | TriggerZones$ Battlefield | Execute$ TrigChangeTarget | TriggerDescription$ Whenever an opponent casts a spell, if it has a single target, change the target to enchanted creature if able.
```

Comment:
@tool4ever does this mean we should remove the Parameter for "IsSingleTarget" ?

Reply:
it is written as Intervening IF Clause

if for example the Opponent cast as Spell that has two or more "target" but each of them target the same target this Aura would trigger.

But if now one of the two or more targets is moved with something like Spellskite while leaving the other targets in tact.(?)
Then when this trigger tries to resolve, it should check the IF Clause again, see that the Spell now has more than a single target and should fizzle.

Reply:
No, check out CR 115.9a

the wordings there are really easy to mix up

this case counts each object

this is also enforced by this ruling as reminder:
> If a spell has multiple targets, it doesn't have a single target even if the same object is chosen for each of those targets.

Reply:
Also this Ruling:
> If a spell or ability targets the same player or object multiple times, you can’t target it with Bolt Bend. (2019-05-03)